### ISSUE TITLE
Bootstrap Offset Classes Missing

### ISSUE THREAD

**Issue by eversionsystems** [holger1411/understrap-child#111]
The offset classes appear to be missing from the npm bootstrap package when I run the install.  I checked the source code files at getbootstrap.com and I can see the classes in bootstrap.css but when I install it via npm they are not there....  I'm using the new format class name "offset-sm-2" which drops the "col-" prefix from bootstrap 3.

I'm new to this process so maybe I've done something wrong....

**Comment by eversionsystems** [comment]
Ok I found the problem, the sccs file **_grid-framework.scss** is missing this declaration.

```
 // `$columns - 1` because offsetting by the width of an entire row isn't possible
      @for $i from 0 through ($columns - 1) {
        @if not ($infix == "" and $i == 0) { // Avoid emitting useless .offset-0
          .offset#{$infix}-#{$i} {
            @include make-col-offset($i, $columns);
          }
        }
      }
```

I downloaded the beta2 version of the boostrap source and I can see it in that.

**Comment by holger1411** [comment]
Hi & thx for your feedback!
Released 0.3.6 today with Beta 2 support. Seems it was a bug in Beta 2.
